src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -94,9 +94,9 @@ const PathwaysList: FC<PathwaysListProps> = ({ evaluatedPathways, callback, serv
       </Button>
 
       <PathwaysTable
-        pathways={evaluatedPathways}
+        pathways={pathways}
         deleteButton={(): void => callback()}
```

Comment:
Since the delete button is not implemented yet can we remove the `callback()` call. This call triggers the screen to change but it isn't passing in a pathway. I think for now we should just have this do nothing
```suggestion
        deleteButton={(): void => {}}
```